KsApi/models/graphql/adapters/GraphMutationWatchProjectResponseEnvelope+WatchProjectMutation.Data.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+import ReactiveSwift
+
+extension GraphMutationWatchProjectResponseEnvelope {
```

Comment:
Can create a new struct called `WatchProjectResponseEnvelope` and extend it here, just like we did for `CreateBackingEnvelope` and `UpdateAccountEnvelope`, that way we can remove dependency on `GraphMutationWatchProjectResponseEnvelope` or in theory any other `Graph` response. Reason being they will be easier to clean up in the future.

Reply:
Hmm am I missing something here. Wouldn't globally renaming the envelope to remove the `GraphMutation` prefix be sufficient here?